modules/file-extension-common/src/main/resources/plugin.properties

```diff
@@ -1,5 +1,7 @@
 artifact.export.resources=
 
+plugin.bundle=org.mule.modules:mule-module-file-extension-common:4.0-SNAPSHOT
```

Comment:
indeed, that's why plugin.properties should be dropped in favour of the POM+maven model, will work on that https://www.mulesoft.org/jira/browse/MULE-11092